projects/ccf-eui/src/environments/environment.prod.ts

```diff
@@ -30,5 +30,6 @@ export const environment = {
     hubmapDataUrl: 'https://search.api.hubmapconsortium.org/entities/search',
     hubmapAssetsUrl: 'https://assets.hubmapconsortium.org',
     hubmapToken: localStorage.getItem('HUBMAP_TOKEN') || ''
-  }
+  },
+  googleAnalyticsToken: 'UA-136932895-2'
```

Comment:
Add staging environment setup. See spoke's environment.stage.ts and surrounding setup. For now use the same GA token, but I'll get you a new one for netlify/staging.